runtime/criusupport/criusupport.cpp

```diff
@@ -376,16 +379,34 @@ Java_org_eclipse_openj9_criu_CRIUSupport_checkpointJVMImpl(JNIEnv *env,
 			goto wakeJavaThreads;
 		}
 
+		/* Iterate heap objects, and run internal hooks before checkpoint - isRestore is FALSE */
+		if (FALSE == vmFuncs->runInternalJVMCriuHooks(vm, FALSE)) {
+			currentExceptionClass = vm->criuSystemCheckpointExceptionClass;
+			nlsMsgFormat = j9nls_lookup_message(J9NLS_DO_NOT_PRINT_MESSAGE_TAG | J9NLS_DO_NOT_APPEND_NEWLINE,
+				J9NLS_JCL_CRIU_FAILED_TO_RUN_INTERNAL_CHECKPOINT_HOOKS, NULL);
+			goto wakeJavaThreadsWithExclusiveVMAccess;
+		}
+
+		Trc_CRIU_before_checkpoint(currentThread);
```

Comment:
also please add tracepoints for entry and exit of this method

Reply:
Added `Trc_CRIU_checkpointJVMImpl_Entry` & `Trc_CRIU_checkpointJVMImpl_Exit`.